reposerver/repository/repository_test.go

```diff
@@ -913,8 +913,8 @@ func TestManifestGenErrorCacheFileContentsChange(t *testing.T) {
 		fmt.Println("    res: ", res)
 
 		if step < 2 {
-			assert.True(t, (err != nil) == errorExpected, "error return value and error expected did not match")
-			assert.True(t, (res != nil) == !errorExpected, "GenerateManifest return value and expected value did not match")
+			assert.Equal(t, errorExpected, (err != nil), "error return value and error expected did not match")
+			assert.Equal(t, (res != nil), !errorExpected, "GenerateManifest return value and expected value did not match")
```

Comment:
I feel like this should be written as `if errorExpected { // assertion here }` instead